gp-includes/formats/format-android.php

```diff
@@ -161,7 +161,8 @@ private function unescape( $string ) {
 	 */
 	protected function escape( $string ) {
 		$string = addcslashes( $string, "'\n" );
-		$string = str_replace( array( '&', '<', '"' ), array( '&amp;', '&lt;', '\"' ), $string );
+		$string = addcslashes( $string, '"' );
```

Comment:
You don't need a second call to addcslashes (it may in fact break things), simply add the double quote to the list of characters to escape (so "'\n" becomes "'\n\"").